app/renderer/components/preferences/syncTab.js

```diff
@@ -137,7 +137,7 @@ class SyncTab extends ImmutableComponent {
     return devices.map((device, id) => [
       {
         html: id,
-        value: parseInt(id)
+        value: device.get('lastRecordTimestamp')
```

Comment:
i don't think is correct because if other devices add records out of order, the id list will be even more jumbled.